readthedocs/projects/models.py

```diff
@@ -449,7 +449,7 @@ def checkout_path(self, version=LATEST):
     @property
     def pip_cache_path(self):
         """Path to pip cache."""
-        if getattr(settings, 'GLOBAL_PIP_CACHE', False):
+        if getattr(settings, 'GLOBAL_PIP_CACHE', False) and settings.DEBUG:
```

Comment:
Is there a default `GLOBAL_PIP_CACHE` in `dev.py`?

Reply:
No. By default this feature is deactivated. It could be something like:

    GLOBAL_PIP_CACHE = os.path.join(SITE_ROOT, '.cache', 'pip')

what do you think?

Reply:
I'd say leave deactivated so our development environments are close to production use.